app/src/main/java/com/android/solvit/provider/ui/profile/ProviderRegistrationProfile.kt

```diff
@@ -183,7 +187,7 @@ fun ProviderRegistrationScreen(
 
   val localContext = LocalContext.current
 
-  val fullNameRegex = Regex("^[A-Za-z]{3,}\\s[A-Za-z]{3,}$")
+  val fullNameRegex = Regex("^[a-zA-Z]+(?:[-' ][a-zA-Z]+)* [a-zA-Z]+$")
```

Comment:
Great improvement on the name regex! Consider extracting this to a shared constants file since it's used in both provider and seeker registration.